docs/deprecations.md

```diff
@@ -6,6 +6,16 @@ layout: Doc
 
 # Serverless Framework Deprecations
 
+<a name="AWS_API_GATEWAY_SPECIFIC_KEYS"><div>&nbsp;</div></a>
+
+## `awsKmsKeyArn` references
+
+Please use `provider.apiGateway.apiKeys` instead of `provider.apiKeys`.
+Please use `provider.apiGateway.resourcePolicy` instead of `provider.resourcePolicy`.
+Please use `provider.apiGateway.usagePlan` instead of `provider.usagePlan`.
+
+Starting with v3.0.0, API Gateway-specific configuration keys "apiKeys", "resourcePolicy" and "usagePlan" will be relocated from "provider" to "provider.apiGateway".
```

Comment:
Minor nitpick: I believe that across this doc we're using backticks instead of "" to highlight specific configuration keys